compiler/src/test-integration/java/io/neow3j/compiler/LedgerContractIntegrationTest.java

```diff
@@ -156,13 +156,11 @@ public void currentHash() throws IOException {
         assertThat(hash.length, is(32));
     }
 
-    @Ignore("Waiting for the implementation of the LedgerContract in the contract module.")
     @Test
     public void getHash() throws IOException {
         NeoInvokeFunction response = callInvokeFunction();
-        // TODO: Get the script hash of the LedgerContract form the contract module.
-//      assertThat(response.getInvocationResult().getStack().get(0).getHexString(),
-//              is(io.neow3j.contract.LedgerContract.SCRIPT_HASH.toString()));
+        assertThat(response.getInvocationResult().getStack().get(0).getHexString(),
+                is("da65b600f7124ce6c79950c1772a36403104f2be"));
```

Comment:
did you use the hex string here on purpose? I think it's better if we start to use something like `LedgerContract.SCRIPT_HASH.toString()`... what do you think?

This applies to other tests.

Reply:
I use `SomeContract.SCRIPT_HASH.toString()` wherever possible. But in this case there is no `LedgerContract` in the `contract` package, therefore, i used the hex string directly.